third_party/subscriptions-project/config.js

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/** Version: 0.1.22.35 */
+/** Version: 0.1.22-1541446533276 */
```

Comment:
This should be the actual release version. Did you run it with the version in command line?

Reply:
Good catch - typo `-swgVersion=` instead of `--swgVersion=` on the command line